src/go/rpk/pkg/cli/cloud/cluster/select.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package cluster
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/cli/profile"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newSelectCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var profileName string
+
+	cmd := &cobra.Command{
+		Use:     "select [NAME]",
+		Aliases: []string{"use"},
```

Comment:
The default should be `use` and `select` an alias... This is to match `rpk cloud auth use` and `rpk profile use`. wdyt?

Reply:
This was first `use`, but `rpk cloud cluster use` without any value looks odd. `use` implies a value is following it. The likely / main flow here is for a person to run `rpk cloud cluster select` and then enter a prompt to select the cluster.

I think select makes more sense given that^ but we can also swap use & select interchangeably between `Use` and `Aliases`, if we need to.

Reply:
Makes sense